src/DogStatsd.php

```diff
@@ -261,7 +261,11 @@ public function send($data, $sampleRate = 1.0, $tags = null)
 
         foreach ($sampledData as $stat => $value) {
             $value .= $this->serialize_tags($tags);
-            $this->report("$stat:$value");
+            if (!is_int($stat)) {
```

Comment:
This doesn't actually fix the issue, since the tags are stored in `$tags` and not in `$sampledData`. Likely a fix for this issue would need to be done in `serialize_tags`..